Project Management Process.md

```diff
@@ -0,0 +1,293 @@
+- Pings each of the discussion leads to remind them to prepare for their talks the next day, preferably by writing an agenda
+- Asks people to drop their updates in the old sprint issue before the sync
+- Reminds people to drop their TODOs in the new sprint issue after the discussions.
+
+** Sync tasks**
+
+- Begins each sprint sync with a roll-call by pinging active contributors (listed below) on IRC.
+- Prompts everyone who participated in the previous sprint to update on their work. The best way to choose who goes first is to go off of a first-post-first-sync method, where all participants add their updates to the old PM issue, and the first to do so generally syncs first in the IRC channel.
+- Closes up by making sure everyone who needs to has gone.
+
+** Discussion tasks**
+
+- Sets up the videos and moderates them, using the process outlined in [hangouts.md](hangouts.md).
+
+## Structure
+
+### Overview
+The project information across the organization is tracked with the structure described in this chapter. At the highest level, we have an Organization Roadmap, which lists all projects and their milestones. Each project contains a set of milestone and milestones contain one or more goals. Goals are the basic unit of work and they describe everything from bugs and new features to general tasks. Goals are collected and tracked in a Backlog.
+
+```
+Organization        |   Project 1           Project 2
+                    |  
+Roadmap             |   Roadmap             Roadmap  
+                    |  
+  Milestones        |     Milestone 1         Milestone A
+    Milestone 1     |       Goal 1              Goal A
+    Milestone 2     |       Goal 2            Milestone B
+    Milestone 3     |       Goal 3              Goal B
+    Milestone A     |     Milestone 2           Goal C
+    Milestone B     |       Goal 4              Goal D
+    Milestone C     |     Milestone 3      
+                    |       Goal 5      
+  Projects          |  
+    Project 1       |   Backlog             Backlog
+    Project 2       |     Goal 1              Goal A
+    Project 3       |     Goal 2              Goal B
+                    |     Goal 3              Goal C
+                    |     Goal 4              Goal D
+                    |     Goal 5
+```
+
+### Goals
+At the heart of each project are the goals. They are the basic unit of work. Goals can be anything that ***adds value to a release***. This includes issues reported by users, feature requests, bugs, Pull-Requests, general tasks such as refactoring, documentation or research. Goals are collected, maintained and tracked in the backlog by the project lead.
+
+A goal description includes the following:
+- Name / Summary
+  - Describes what the goal is on high level
+  - Example 1: "Reduce the size of the go-ipfs executable"
+  - Example 2: "js-ipfs should have API Documentation"
+- Description
+  - What the problem is and how to achieve this goal
```

Comment:
> one thing to consider here is not over-detailing, meaning every detail we add will take time out of every person and every thing they do. if the distinction is not necessary or very important let's not force it.

This is hugely important point. I don't think *forcing* anything, be it a template or otherwise, will help us get off the ground. A lot of the "best practices" will come over time as we do it and get a feeling of what matters and what doesn't. And individually we'll become better at wording them as we enter new goals and tasks for our projects (eg. how to word a task as a goal but still have a "task" in the backlog).